src/main/java/Utils/messageHandler.java

```diff
@@ -0,0 +1,150 @@
+package Utils;
+
+import Entity.Message;
+import Entity.messageList;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.Scanner;
+
+public class messageHandler {
+
+    public static void preHandle(messageList list) {
```

Comment:
Maybe using a more descriptive name such as preHandleInput might be better here to specify what this method is doing.